web/views/view_helpers.ex

```diff
@@ -0,0 +1,64 @@
+defmodule App.ViewHelpers do
+  @moduledoc """
+    Functions to transform html as stored in the database by wagtail into
+    the html we actually render on the site
+  """
+
+  import App.Router.Helpers
+  import Ecto.Query, only: [from: 2]
+
+  alias App.CMSRepo
+
+  # Regex to match wagtail embedded image and pull out title, id and format
+  @embed_regex ~r/<embed\salt=\"([^\"]+)\"\sembedtype=\"image\"\sformat=\"([^\"]+)\"\sid=+\"([^\"]+)\"\/>/
+  @link_regex ~r/<a\sid=\"([^\"]+)\"\slinktype=\"page\">([^<]+)<\/a>/
+
+  def renderImage(html_string, conn) do
```

Comment:
The conn is usually used as the first argument because that's what's being changed and passed around. In these functions I'm piping the html string from one function to another and editing it (see https://github.com/LDMW/app/pull/125/files/3a93366a28b0c854a6a261deb0eb97f20c20a328#diff-2eea8c23e7a51d054805c8e62035f5e9R12) so the html string has to be first